tests/e2e/editor-selector.spec.js

```diff
@@ -0,0 +1,38 @@
+const {test, expect} = require('@playwright/test');
+import {login} from './tools/lib/login';
+
+test('Test the creation of a new Post with an embedded video and paragraph and check if they are present when published', async ({page, context}) => {
```

Comment:
This is a really long test name, could we shorten it? Maybe `Test the creation of a new Post` is enough 🤔